contracts/Example2.sol

```diff
@@ -1,11 +1,16 @@
 pragma solidity ^0.8.4;
 
+// Adding with sum reduces gas from 50355 to 49467
+// Not enough yet!
+
 contract Example2 {
     uint public counter;
  
-    function incrementBy(uint[] memory arr) public {
+    function incrementBy(uint[] calldata arr) public {
```

Comment:
This can be an external function

Reply:
Thanks. I don't see any gas cost difference vs. only changing to calldata.